Submitty/Submitty
# [UI/UX:Submission] Total results style change

### This PR
1. Eliminates the overview section introduced in #5085 
2. Removes section titles within each island and modifies appearance of box containing grading totals.

### To test:
1. View results of peer, ta, and autograding and their permutations (ta but no autograding, autograding but no ta grading, etc.)
2. Check hidden testcase output before and after grades have been released.
3. Check confetti.

<img width="913" alt="total-style-change-1" src="https://user-images.githubusercontent.com/18623979/75621147-348e8e80-5b5f-11ea-9c65-33e830f5ac6e.png">
<img width="912" alt="total-style-change-2" src="https://user-images.githubusercontent.com/18623979/75621148-348e8e80-5b5f-11ea-847b-6fa0627b9c58.png">
